### ISSUE TITLE
FIX UNDISCORD SCRUB!

### ISSUE THREAD

**Issue by realBernieReid** [victornpb/keyboardly#9]
### Hey! If you could drag yourself away from whatever pointless distraction you're on and actually get back to work on the Undiscord extension, that'd be fantastic. Thanks!

**Comment by Lyall-A** [comment]
yeah what the fuck are you contributing to this platform then scrub? you spelled helicopter wrong in your profile btw
